apps/hash-api/src/graph/ontology/primitive/data-type.ts

```diff
@@ -98,7 +97,7 @@ export const getDataTypeById: ImpureGraphFunction<
         },
       },
     })
-    .then(({ data }) => mapSubgraph(data) as Subgraph<DataTypeRootType>);
+    .then(({ data }) => data as Subgraph<DataTypeRootType>);
```

Comment:
```suggestion
    .then(({ data: subgraph }) =>
      getRoots(subgraph as Subgraph<DataTypeRootType>),
    );
```

Reply:
Applied in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)